internal/api/resolver_query_find_studio.go

```diff
@@ -8,6 +8,10 @@ import (
 )
 
 func (r *queryResolver) FindStudio(ctx context.Context, id string) (ret *models.Studio, err error) {
+	if id == "" {
+		return nil, nil
+	}
```

Comment:
This is not consistent with the behaviour of the other resolvers. An empty id should be an error condition.